packages/cli/src/oclif/commands/cache/clear.js

```diff
@@ -0,0 +1,111 @@
+const BaseCommand = require('../../ZapierBaseCommand');
+const { buildFlags } = require('../../buildFlags');
+const { listVersions, getWritableApp, callAPI } = require('../../../utils/api');
+const { cyan } = require('colors/safe');
+
+class ClearCacheCommand extends BaseCommand {
+  async perform() {
+    this.startSpinner('Fetching versions...');
+    const { versions } = await listVersions();
+    this.stopSpinner();
+
+    const { majorVersion } = this.args;
+
+    let selectedMajorVersion =
+      majorVersion === '' ? null : Number(majorVersion);
+
+    const majorVersions = [
+      ...new Set(
+        versions.map((appVersion) => Number(appVersion.version.split('.')[0]))
+      ),
+    ];
+    // Finds the current version in package.json.
+    const currentVersion = await require(`${process.cwd()}/package.json`)
+      .version;
+
+    if (selectedMajorVersion === null) {
+      selectedMajorVersion = await this._promptForMajorVersionSelection(
+        majorVersions,
+        currentVersion
+      );
+    } else {
+      if (!majorVersions.includes(selectedMajorVersion)) {
+        throw new Error(
+          `This app does not have any versions on major version '${selectedMajorVersion}'. Valid versions are: ${majorVersions.join(
+            ', '
+          )}`
+        );
+      }
+    }
+
+    if (
+      !(await this.confirm(
+        `Are you sure you want to clear the app cache for major version '${cyan(
+          selectedMajorVersion
+        )}'?`,
+        true
+      ))
+    ) {
+      this.log('\ncancelled');
+      return;
+    }
+
+    this.startSpinner('Clearing cache');
+    const { id: appId } = await getWritableApp();
+    const url = `/apps/${appId}/major-versions/${selectedMajorVersion}/cache`;
+
+    await callAPI(url, {
+      url: url.startsWith('http') ? url : undefined,
+      method: 'DELETE',
+    });
```

Comment:
Looks like the `url.startsWith('http')` will always be false and unnecessary?

```suggestion
    await callAPI(url, { method: 'DELETE' });
```